storage/block/block.go

```diff
@@ -36,7 +37,10 @@ var (
 	errReadFromClosedBlock         = errors.New("attempt to read from a closed block")
 	errRetrievableBlockNoRetriever = errors.New("attempt to read from a retrievable block with no retriever")
 
-	timeZero = time.Time{}
+	timeUnixNanoZero = int64(math.MinInt64)
+	// NB(cw): the behavior of calling UnixNano() on time.Time{} is undefined
+	// using this timeZero makes testing a bit easier.
+	timeZero = time.Unix(0, timeUnixNanoZero)
```

Comment:
I kinda want to make timeUnixNanoZero and timeZero interchangeable, gonna do:
	timeZero         = time.Unix(0, 0)
	timeUnixNanoZero = timeZero.UnixNano()